### PR TITLE
Add some content to the page about running the kubevirt demo

### PR DESCRIPTION
**Pull request by berrange**

### CODE REVIEW

File: running.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+provides an all-in-one Kubernetes & KubeVirt installation in a KVM compatible
+disk image.
+
+### Running automatically
+
+The manual steps listed in the next section can be performed automatically
+by running:
+
+```shell
+ $ curl http://run.kubevirt.io/demo.sh | bash
```
Comment (fabiand) [inline running.md]: We _really_ need to update this demo to use minikube.

but this requires libvirt in pod networking.

Comment (berrange) [inline running.md]: Sure, but I don't think that's really important from pov of this PR. This demo is the kind of thing we're going to want to update every few months as the code evolves regardless.

**Comment by scollier** [comment]
@fabiand this still valid?

**Comment by fabiand** [comment]
No

**Pull request closed without merging**